docs/02-app/01-building-your-application/10-deploying/03-multi-zones.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Multi Zones
+description: Learn how to build micro-frontends using Next.js Multi Zones to deploy multiple Next.js apps under a single domain.
+---
+
+{/* The content of this doc is shared between the app and pages router. You can use the `<PagesOnly>Content</PagesOnly>` component to add content that is specific to the Pages Router. Any shared content should not be wrapped in a component. */}
+
+<details open>
+  <summary>Examples</summary>
+
+- [With Zones](https://github.com/vercel/next.js/tree/canary/examples/with-zones)
+
+</details>
+
+Multi Zones are an approach to micro-frontends that separate a single large application on a domain into smaller applications that each serve a set of paths. This is useful when there are collections of pages unrelated to the other pages in the application. By moving those pages to a separate zone, you can reduce the size of each application which improves build times and removes code that is only necessary for one of the zones.
+
+For example, let's say you have the following set of pages that you would like to split up:
+
+- `/blog/*` for all blog posts
+- `/docs/*` for all documentation
+- The rest of your website
```

Comment:
Would it be too confusing to have the 3rd bullet be ``/*` for the rest of your website`

Reply:
Done.